### PR TITLE
Add support for math text to slider widgets

### PR DESCRIPTION
**Pull request by IuryPiva**
image

- [X] issues: relates to #11355  and https://github.com/holoviz/panel/issues/1324
- [x] tests added / passed

### CODE REVIEW

File: bokehjs/src/lib/models/widgets/abstract_slider.ts [hunk 1]
```diff
@@ -153,6 +159,7 @@ abstract class AbstractBaseSliderView extends OrientedControlView {
 
     this.group_el = div({class: inputs.input_group}, this.title_el, this.slider_el)
     this.el.appendChild(this.group_el)
+    this._has_finished = true
```
Comment (mattpap) [inline bokehjs/src/lib/models/widgets/abstract_slider.ts:162]: Why is this necessary?

Comment (IuryPiva) [reply bokehjs/src/lib/models/widgets/abstract_slider.ts:162]: Because we set it to false if mathjax is still loading after first layout
https://github.com/bokeh/bokeh/blob/40ca075d40f951ec604d61be8ba43ddffbc17c5e/bokehjs/src/lib/models/widgets/widget.ts#L55

**Comment by bryevdv** [comment]
What's remaining for this PR to be merged?

**Comment by mattpap** [comment]
> What's remaining for this PR to be merged?

The overall design and pollution of `WidgetView`.

**Comment by bryevdv** [comment]
> pollution of WidgetView

Absent multiple inheritance / mixin classes for latex capability, what is your proposed alternative to a wide API on the base class?

**Comment by IuryPiva** [comment]
> The overall design and pollution of `WidgetView`.

I have made an experiment to move all the stuff related to TeX into a self contained module. #11843 

 Since we don't need to deal with async image loading like we do with canvas rendering we don't need to manage the `_has_finished` property or the lifecycles of views. It's possible to use simple methods to process the math into an HTML string with SVGs

**Comment by mattpap** [comment]
I will go ahead and merge this for the sake of adding examples in PR #11858, but the code will have to be refactored later-on to avoid namespace pollution.
